.github/workflows/people.yml

```diff
@@ -71,34 +71,76 @@ jobs:
         with:
           node-version: '18.x'
           cache: 'yarn'
-          cache-dependency-path: src/frontend/app/desk/yarn.lock
+          cache-dependency-path: src/frontend/yarn.lock
       - name: Install dependencies
         run: yarn install --frozen-lockfile
       - name: Test Desk App
         env:
           CI: true
         run: yarn test
 
-  build-front-desk:
+  test-e2e:
     runs-on: ubuntu-latest
-    env:
-      NEXT_PUBLIC_API_URL: http://localhost:8071/api/v1.0/
-    defaults:
-      run:
-        working-directory: src/frontend/app/desk
+    
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        
+      - name: Set services env variables
+        run: |
+          cp env.d/development/common.dist env.d/development/common
+          cp env.d/development/postgresql.dist env.d/development/postgresql
+          cp env.d/development/kc_postgresql.dist env.d/development/kc_postgresql
+          cp env.d/development/crowdin.dist env.d/development/crowdin
+
+      - name: Start Docker Servers
+        run: |
+          # Check if Docker Layer Caching is available
+          if [[ -z "${{ runner.docker }}" ]]; then
+            echo "Docker Layer Caching is not available. Building without caching."
+          else
+            # Will restore the cached Docker layers, if available
+            docker-compose pull --ignore-pull-failures
+            docker-compose build --pull
+          fi
+      
+          # Start the services needed for the tests
+          docker-compose up -d nginx
+          docker-compose up -d app-dev
+          docker-compose up -d celery-dev
+          docker-compose up -d keycloak
+      
+          make migrate
+        env:
+          DOCKER_BUILDKIT: 1
+
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
           node-version: '18.x'
           cache: 'yarn'
-          cache-dependency-path: src/frontend/app/desk/yarn.lock
+          cache-dependency-path: src/frontend/yarn.lock
+
       - name: Install dependencies
-        run: yarn install --frozen-lockfile
-      - name: Build Desk App
-        run: yarn build
+        run: cd src/frontend/ && yarn install --frozen-lockfile
+
+      - name: Install Playwright Browsers
+        run: cd src/frontend/ && yarn e2e:install
+
+      - name: Build CI App
+        run: cd src/frontend/ && yarn ci:build
```

Comment:
I might not sure why we have a dedicated CI build

Reply:
Because we want to build but with some special `.env`, keycloak is working only in dev by exemple, so to make it works with the build, we need to add some variables at build time, this variables at build time we want them only with the CI, in production these env will be different.